### ISSUE TITLE
Dropdown examples showing icons as toggles

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1837]
This is just an update to the examples, and not the component itself.

Whenever an icon is used as the button label, like the kebab in one of the Dropdown examples, `aria-label` should be included to provide accessible text for the toggle button.

For our Dropdown examples, there are two updates required:

1. For the **`<KebabToggle>`** component, include a default string "Actions" for `aria-label` unless `aria-label` is specified on the component. 
    - In our example, we would still have the toggle defined as:
      `<KebabToggle onToggle={this.onToggle} />`
    - The documentation for `KebabToggle` should include `aria-label` showing that the default value is "Actions"
2. For the example  **Dropdown (icon only)**, `aria-label` on the toggle component so that the component is defined in this example like this: 
    `<DropdownToggle iconComponent={null} onToggle={this.onToggle} aria-label="Applications">`

@tlabaj for PR review

**Comment by jgiardino** [comment]
As part of this issue, for the `<KebabToggle>` component, can we have a default string "Actions" for `aria-label` unless this is specified on the component? If that is the case, then we can just define the component in our example as:
`<KebabToggle onToggle={this.onToggle} >` // without the `aria-label`  

I'm suggesting this for `KebabToggle` only and not `DropdownToggle`, since `KebabToggle` is always an icon, and therefore should always have a value for `aria-label`. But `DropdownToggle` could be text or icon, and `aria-label` should only be specified if a text string is not used as the toggle label.
